src/Uplink/View/Provider.php

```diff
@@ -13,7 +13,7 @@ final class Provider extends Abstract_Provider {
 	public function register() {
 		$this->container->singleton(
 			WordPress_View::class,
-			new WordPress_View( __DIR__ . '/../../views' )
+			new WordPress_View( __DIR__ . '/../../views', '.php' )
```

Comment:
This already defaults to `.php`
https://github.com/stellarwp/uplink/blob/f0f06102ffc265071965af183b1a31e025096910/src/Uplink/View/WordPress_View.php#L36